js/map_utils.js

```diff
@@ -115,6 +118,10 @@ function updateView(newView, fireAnalytics = true) {
       nationDims.width/stateDims.width]);
   }
 
+  // update the geospatial data for the upcoming resolution
+  updateCounties(activeView);
+  updateStates(activeView);
+  
```

Comment:
I can't add a comment there, but on a line below within `updateView`, your merge conflicts I think kept these lines and shouldn't have. That's wrapped up in `backgroundState` and `foregroundState` functions but having it here is overriding those.

```
statecounties
      .transition()
      .duration(500) 
      .style("stroke-width",  0.75/zoom_scale); // make all counties have scaled stroke-width
```

Reply:
so just delete the lines you quoted, right?

Reply:
hmm - are the county lines supposed to be giant now?
![image](https://user-images.githubusercontent.com/12039957/38337592-c796dc38-381a-11e8-8cac-a0354044e1ec.png)

Reply:
now we have

![image](https://user-images.githubusercontent.com/12039957/38338012-93a77eda-381c-11e8-940a-a65213a411f4.png)

which looks more like what i expected/wanted

Reply:
But what I'm not seeing is the state border getting much thicker than the background state borders....

Reply:
I expect this, where foreground state is bold and background states aren't. I should've looked more closely at the code I suggested to throw out, but there is still something different

![image](https://user-images.githubusercontent.com/13220910/38370652-39f34970-38b0-11e8-8ed0-48f567b7cf0e.png)